web/styles/portico/hello.css

```diff
@@ -1012,6 +1012,7 @@ ul {
             transform: scale(0.6);
             grid-template-columns: repeat(3, 200px);
             grid-template-rows: 100px 100px;
+            margin: -29px -500px;
```

Comment:
I always worry that anything like a -500px margin will end up failing eventually, but I guess this has a pretty limited rang of widths, and it's a static landing so maybe we can live with it.

Reply:
Yeah, I am not happy with it either, but the alternative is having a lot constants sizes for this window width for the images and the space between and around them., which is not easy to maintain either.